test/games/prime2/patcher/test_patch_data_generator.py

```diff
@@ -708,47 +717,11 @@ def test_generate_patcher_data(test_files_dir, use_new_patcher):
     )
 
     # Assert
-    assert isinstance(result["spawn_point"], dict)
+    expected_results_path = test_files_dir.joinpath("patcher_data", "prime2", expected_results_filename)
 
-    assert isinstance(result["pickups"], list)
-    assert len(result["pickups"]) == 119
+    # Uncomment to easily view diff of failed test
+    json_lib.write_path(expected_results_path, result)
```

Comment:
Without `#` it's cheating and tests will always pass :p

Reply:
oops. fixed